.tekton/central-db-build.yaml

```
@@ -8,13 +8,16 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "500"
-    # TODO(ROX-21073): re-enable for all PR branches
     pipelinesascode.tekton.dev/on-cel-expression: |
-      (event == "push" && target_branch.matches("^(master|release-.*)$")) ||
-      (event == "pull_request" && (source_branch.matches("(konflux|appstudio|rhtap)") || body.pull_request.labels.exists(l, l.name == "konflux-build")))
+      event == "pull_request" || (
+        event == "push" && (
+          target_branch.startsWith("release-") ||
```

I think
```
          target_branch.startsWith("release-") ||
```
still causes double-triggering in this PR since it's both a PR event and pushed commits to it _target_ `release-4.6` branch.
I'm 90% sure there is no double-triggering but could you please still check if that's the case?

I think, the solution would be
```suggestion
          source_branch.startsWith("release-") ||
```

---

That makes sense.  target_branch == source_branch for non-PR pushes, so I don't see any harm in going ahead and making this change.